storage/innobase/os/os0file.cc

```diff
@@ -8709,7 +8709,6 @@ void Encryption::get_master_key(ulint *master_key_id, byte **master_key) {
   memset(key_name, 0x0, sizeof(key_name));
 
   if (s_master_key_id == 0) {
-    ut_ad(strlen(server_uuid) > 0);
```

Comment:
This removal looks suspicious

Reply:
This check was added by 4d747cf709910ae0b6ed4626b0fca843c37d10e1 : Bug#29006275: ENCRYPTION MASTER KEY IS GENERATED WITH BLANK UUID. We have a better fix for it with fix_empty_uuid handler - thus I removed this. I should have added this to the commit message and will do that now.

Reply:
ok, thnx